.github/workflows/cypress-tests-runner.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 env:
   CARGO_INCREMENTAL: 0
   CARGO_NET_RETRY: 10
-  PAYMENTS_CONNECTORS: "cybersource stripe"
+  PAYMENTS_CONNECTORS: "archipel cybersource stripe"
```

Comment:
why do we want to run archipel against all prs? all the tests pass? if not, this will stop every pr from merging.